app/src/main/java/com/arctouch/codechallenge/home/data/cache/GenresCache.kt

```diff
@@ -0,0 +1,12 @@
+package com.arctouch.codechallenge.home.data.cache
+
+import com.arctouch.codechallenge.core.domain.model.Genre
+
+object GenresCache {
+
+    var genres: List<Genre>? = null
```

Comment:
Mainly the encapsulation principle, it´s a issue because anywhere can access and change this var, potentially leading to errors, security issues and other problems